components/dashboard/sections/contributions/Contributions.tsx

```diff
@@ -130,21 +227,70 @@ const dashboardContributionsQuery = gql`
   ${managedOrderFragment}
 `;
 
-const getColumns = ({ tab, setEditOrder, intl, isIncoming }) => {
+const getColumns = ({ tab, intl, isIncoming, includeHostedAccounts, onlyExpectedFunds }) => {
+  const accounts = {
+    accessorKey: 'toAccount',
+    header: intl.formatMessage({ defaultMessage: 'Collective & Contributors', id: 'kklCrk' }),
+    meta: { className: 'max-w-[200px] overflow-hidden' },
+    cell: ({ cell, row }) => {
+      const toAccount = cell.getValue();
+      const fromAccount = row.original.fromAccount;
+      return (
+        <div className="flex items-center gap-5">
+          <div className="relative">
+            <div>
+              <AccountHoverCard
+                account={toAccount}
+                trigger={
+                  <span>
+                    <Avatar size={32} collective={toAccount} displayTitle={false} />
+                  </span>
+                }
+              />
+            </div>
+            <div className="absolute -bottom-[6px] -right-[6px] rounded-full">
+              <AccountHoverCard
+                account={fromAccount}
+                trigger={
+                  <span>
+                    <Avatar size={16} collective={fromAccount} displayTitle={false} />
+                  </span>
+                }
+              />
+            </div>
+          </div>
+          <div className="overflow-hidden">
+            <div className="overflow-hidden text-ellipsis whitespace-nowrap text-sm font-medium leading-5">
+              {toAccount.slug || toAccount.name}
+            </div>
+            <div className="overflow-hidden text-ellipsis whitespace-nowrap text-xs leading-4">
+              {fromAccount.slug || fromAccount.name}
+            </div>
```

Comment:
I think we should prioritize name here, it looks especially weird with vendors.
```suggestion
            <div className="overflow-hidden text-ellipsis whitespace-nowrap text-sm font-medium leading-5">
              {toAccount.name || toAccount.slug}
            </div>
            <div className="overflow-hidden text-ellipsis whitespace-nowrap text-xs leading-4">
              {fromAccount.name || fromAccount.slug}
            </div>
```

Reply:
![image](https://github.com/opencollective/opencollective-frontend/assets/2119706/f7651431-d17c-446c-91f2-de2c94e108d7)